java/org/apache/tomcat/jni/Library.java

```diff
@@ -247,7 +247,7 @@ public static synchronized boolean initialize(String libraryName) throws Excepti
             APR_CHARSET_EBCDIC      = has(18);
             APR_TCP_NODELAY_INHERITED = has(19);
             APR_O_NONBLOCK_INHERITED  = has(20);
-            APR_HAVE_UNIX             = has(22);
+            APR_HAVE_UNIX             = has(21);
```

Comment:
Is this 21 reflected in native code?

Reply:
...and that would be where the 21 comes from:

```
        case 21:
#if defined(APR_POLLSET_WAKEABLE)
            rv = JNI_TRUE;
#endif
        break;
```

Let me revert.

Reply:
Done.